src/data/contributors/Aditya.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Aditya Rauthan",
+  "skills": ["Python", "Django", "Bootstrap"],
+  "cause": "tech advancement",
+  "image": "Avatar3",
+  "github": "https://github.com/Adityarauthan2409",
+  
+  "linkedin": "https://www.linkedin.com/in/aditya-rauthan"
```

Comment:
Hey @Adityarauthan2409 , Seems like you have messed up with your linked profile link.
![image](https://user-images.githubusercontent.com/48255244/88849359-6176c880-d207-11ea-84b8-1ade1a4e84f7.png)
The url pointing to the profile is not available. please correct it.